Backend/DataAccessLayer/DataAccessLayer.csproj

```diff
@@ -1,131 +1,138 @@
+    <Compile Include="Migrations\201903130042345_update-tables-passwords.Designer.cs">
+      <DependentUpon>201903130042345_update-tables-passwords.cs</DependentUpon>
+    </Compile>
+    <Compile Include="Migrations\Configuration.cs" />
+    <Compile Include="Models\ApiKey.cs" />
+    <Compile Include="Models\Application.cs" />
+    <Compile Include="Models\LoginRequest.cs" />
+    <Compile Include="Models\Session.cs" />
+    <Compile Include="Models\User.cs" />
+    <Compile Include="Properties\AssemblyInfo.cs" />
+    <Compile Include="Repositories\ApiKeyRepository.cs" />
+    <Compile Include="Repositories\ApplicationRepository.cs" />
+    <Compile Include="Models\PasswordReset.cs" />
+    <Compile Include="Repositories\ResetRepository.cs" />
+    <Compile Include="Repositories\SessionRepository.cs" />
+    <Compile Include="Repositories\UserManagementRepository.cs" />
+    <Compile Include="Repositories\UserRepository.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <None Include="App.config">
+      <SubType>Designer</SubType>
+    </None>
+    <None Include="packages.config" />
+  </ItemGroup>
+  <ItemGroup>
+    <Service Include="{508349B6-6B84-4DF5-91F0-309BEEBAD82D}" />
+  </ItemGroup>
+  <ItemGroup>
+    <EmbeddedResource Include="Migrations\201812040636053_create-table-user.resx">
+      <DependentUpon>201812040636053_create-table-user.cs</DependentUpon>
+    </EmbeddedResource>
+    <EmbeddedResource Include="Migrations\201812040642468_create-table-session.resx">
+      <DependentUpon>201812040642468_create-table-session.cs</DependentUpon>
+    </EmbeddedResource>
+    <EmbeddedResource Include="Migrations\201903070039162_create-tables-applications-apikeys.resx">
+      <DependentUpon>201903070039162_create-tables-applications-apikeys.cs</DependentUpon>
+    </EmbeddedResource>
+    <EmbeddedResource Include="Migrations\201903090835106_create-table-passwordreset.resx">
+      <DependentUpon>201903090835106_create-table-passwordreset.cs</DependentUpon>
+    </EmbeddedResource>
+    <EmbeddedResource Include="Migrations\201903100743056_create-attribute-incorrectPasswordCount.resx">
+      <DependentUpon>201903100743056_create-attribute-incorrectPasswordCount.cs</DependentUpon>
+    </EmbeddedResource>
+    <EmbeddedResource Include="Migrations\201903110756108_ghostMigration.resx">
+      <DependentUpon>201903110756108_ghostMigration.cs</DependentUpon>
+    </EmbeddedResource>
+    <EmbeddedResource Include="Migrations\201903111003125_upate-sessions-userNotUnique.resx">
+      <DependentUpon>201903111003125_upate-sessions-userNotUnique.cs</DependentUpon>
+    </EmbeddedResource>
+    <EmbeddedResource Include="Migrations\201903130042345_update-tables-passwords.resx">
+      <DependentUpon>201903130042345_update-tables-passwords.cs</DependentUpon>
+    </EmbeddedResource>
+  </ItemGroup>
+  <ItemGroup />
+  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />

```

Comment:
Why is this whole file modified? Please revert this and make only the changes you need.